doc/ale-precommit.txt

```diff
@@ -0,0 +1,32 @@
+===============================================================================
+ALE pre-commit Integration                                           *ale-precommit-options*
+
```

Comment:
Align the documentation tags to the right margin. `:ALEFix align_help_tags`.

Reply:
ah jeez, I was fighting with spaces forever on this one 😬 . This is much better